numpy/core/fromnumeric.py

```diff
@@ -2516,6 +2520,17 @@ def cumsum(a, axis=None, dtype=None, out=None):
     array([[ 1,  3,  6],
            [ 4,  9, 15]])
 
+    ``cumsum(b)[-1]`` may not be equal to ``sum(b)``
+
+    >>> b = np.array([1, 2e-9, 3e-9] * 1000000)
+    >>> np.cumsum(b)
+    array([1.00000000e+00, 1.00000000e+00, 1.00000000e+00, ...,
+           1.00000001e+06, 1.00000001e+06, 1.00000001e+06])
+    >>> b.cumsum()[-1]
+    1000000.0050045159            # summation using cumsum() 
+    >>> b.sum()                    
+    1000000.0050000029            # summation using sum()
```

Comment:
```suggestion
    1000000.0050045159
    >>> b.sum()                    
    1000000.0050000029
```

The comments in the output lines are unnecessary, it should be obvious what the values represent.